### PR TITLE
Combine `@xstate/graph` and `@xstate/test`

### PR DESCRIPTION
**Pull request by davidkpiano**

### CODE REVIEW

**Comment by davidkpiano** [comment]
@Andarist When you get the chance, can you set this up so that it will do a proper 2.0 release?

**Comment by davidkpiano** [comment]
@Andarist Ping 🏓

File: .changeset/eighty-papayas-exist.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/graph': patch
+---
+
+The `@xstate/graph` package now includes everything from `@xstate/test`.
```
Comment (Andarist) [inline .changeset/eighty-papayas-exist.md:5]: What are the current roles of both packages? I guess that with the "combine" in the title of this PR I expected this one would consolidate those 2 in a single package but we still have 2

File: packages/xstate-graph/src/TestModel.ts
Comment (Andarist) [inline packages/xstate-graph/src/TestModel.ts:1]: I diffed this and the other one in xstate-test. They are **very very** similar but yet they have some very small differences. I'm confused 😅

File: .changeset/config.json [hunk 2]
```diff
@@ -5,12 +5,7 @@
   "linked": [],
   "access": "public",
   "baseBranch": "main",
-  "ignore": [
```
Comment (Andarist) [inline .changeset/config.json:8]: q: do u plan to release `@xstate/graph` as a stable release **soon**? we probably should add `@xstate/graph` back here and do some cleanup in https://github.com/statelyai/xstate/tree/7c6e2eaa4e5021738efec345b51f59a8e2de662e/.changeset-xstate-test-beta and only then execute the final release

File: packages/xstate-graph/test/shortestPaths.test.ts [hunk 3]
```diff
@@ -131,7 +131,7 @@ describe('getShortestPaths', () => {
           todo: 'two'
         } as const
       ],
-      filter: (state) => state.context.todos.length < 3
+      stopWhen: (state) => state.context.todos.length >= 3
```
Comment (Andarist) [inline packages/xstate-graph/test/shortestPaths.test.ts:134]: this feels like a breaking change that wasn't mentioned in changesets

Comment (davidkpiano) [reply packages/xstate-graph/test/shortestPaths.test.ts:134]: I will add a changeset in a follow-up (clean-up) PR

File: .changeset/config.json (hunk 2 above)
Comment (davidkpiano) [reply .changeset/config.json:8]: Yes, I plan to release it soon

File: packages/xstate-graph/test/shortestPaths.test.ts (hunk 3 above)
Comment (davidkpiano) [reply packages/xstate-graph/test/shortestPaths.test.ts:134]: #4911

File: .changeset/config.json (hunk 2 above)
Comment (davidkpiano) [reply .changeset/config.json:8]: #4911

File: packages/xstate-graph/src/TestModel.ts
Comment (davidkpiano) [reply packages/xstate-graph/src/TestModel.ts:1]: Changesets in #4911

File: .changeset/eighty-papayas-exist.md (hunk 1 above)
Comment (davidkpiano) [reply .changeset/eighty-papayas-exist.md:5]: Deleted `@xstate/test` in #4911

**Pull request merged**